### ISSUE TITLE
More detail in README please

### ISSUE THREAD

**Issue by RavenousPett** [MathildeLemee/hubot-twitter#21]
"Then you will need to set the HTTP endpoint on Twitter to point to your server and make sure the request type is set to GET"

I'm not exactly sure where to go on Twitter to set this. Please can you help? I'm deploying with Heroku.

**Comment by dnvikram** [comment]
Indeed, need of the hour is a detailed readme to help people jumpstarted with this module.

**Comment by MathildeLemee** [comment]
Do not hesitate to send a pull request to improve that point. 

     Le Jeudi 15 janvier 2015 21h47, Narendra Vikram D <notifications@github.com> a écrit :
   

 Indeed, need of the hour is a detailed readme to help people jumpstarted with this module.—
